cds/modules/records/static/templates/cds_records/detail.default.html

```diff
@@ -2,35 +2,104 @@
   <div class="col-md-12">
     <div class="cds-record-detail">
       <div class="cds-record-detail-top">
-        <ul class="list-inline">
+        <!-- <ul class="list-inline">
           <li><a href="#">Metadata</a></li>
           <li><a ng-show="record.files.length > 0" href="#files">Files</a></li>
           <li><a href="#export">Export</a></li>
-        </ul>
+        </ul> -->
```

Comment:
Yes, I wanted to actually ask first if we are sure we want to remove those `tabs`